.hound.yml

```diff
@@ -10,9 +10,4 @@ ruby:
 jshint:
   enabled: false
 
-eslint:
-  enabled: true
```

Comment:
is it disable by default? maybe make explicit enabled:false ?

Reply:
It is disabled by default and `jshint` is enabled by default